CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+> Release Date: future
+
+Enhancement:
```

Comment:
```suggestion
Enhancements:

```

Reply:
addressed in (131acd4bec5c68a803eb55ef8c6b8d2638b81d17)